### DISCUSSION TITLE
Deprecation Notice: Packaging Changes in 1.29

### DISCUSSION THREAD

**Discussion by haircommander** [cri-o/cri-o#7315] [Announcements]
Hello all!

The CRI-O team is excited to announce two packaging changes that are starting in 1.29. 
- The first was [previously announced](https://github.com/cri-o/cri-o/issues/6986) and involves removing the fedora modules in favor of a native fedora package. The changes will begin in F39, which corresponds timeline-wise to CRI-O 1.29. Thanks to @LorbusChris for spearheading this effort!
- With the advent of [pkgs.k8s.io](https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/) and imminent freeze of the legacy packages, the CRI-O team (primarily @saschagrunert :tada: ) has been at work [moving our OBS footprint](https://github.com/cri-o/cri-o/issues/7280) over to the `isv:kubernetes` project. More info on this change is in https://github.com/cri-o/packaging and the installation instructions will be updated before the release. To ease transition, we've already begun packaging [1.28](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.28) 

Let us know if you have any questions or any desires out of the new packaging architecture. We hope the transition is seamless and involves less overall work for the maintainers.

**Comment by tnorlin** [comment]
I have a question on this for the CRI-O 1.29 as I find the instructions in the home page eventually to  be deprecated?

The front page instructions refers to packages at https://download.opensuse.org/../ )? But in the cri-o/packaging repo I find this one instead, which seem to reflect the announcement at the kubernetes.io/blog as well as the above announcement: https://github.com/cri-o/packaging?tab=readme-ov-file

Is that the most recent/accurate instruction for now as it points to https://pkgs.k8s.io (and for APT-based distros have removed the cri-o-runc) ?

**Reply by saschagrunert** [reply]
Yes, you're right the information is outdated. Are you considering to work on a PR to fix that?
